### ISSUE TITLE
config/Fedora/*

### ISSUE THREAD

**Issue by xsuchy** [rpm-software-management/rpmlint#619]
Is there some reason why `config/Fedora/*` does not exist? Is someone working on it? Or is PR welcome?

**Comment by marxin** [comment]
During the development, we were told that `Fedora` (as opposed to `openSUSE`) uses default settings.
If you have a setting overrides, feel free to submit corresponding config files.

**Comment by xsuchy** [comment]
When I run the development code on my package I get:
```
copr-frontend.src: W: strange-permission copr-frontend-1.175.tar.gz 664
copr-frontend.src: W: strange-permission copr-frontend.spec 664
copr-frontend.src: E: no-signature
copr-frontend.src: E: no-packager-tag
copr-frontend.src: E: no-group-tag
copr-frontend.spec: W: no-cleaning-of-buildroot %clean
copr-frontend.spec: W: no-buildroot-tag
copr-frontend.src: W: invalid-license GPLv2+
```

These are all false negatives for Fedora. I.e. all of them should be fine.
Should I submit PR to change the defaults or should I create the config file?

**Comment by marxin** [comment]
> Should I submit PR to change the defaults or should I create the config file?

Create a separate config file(s), please.
